app/pages/project/instances/InstancesPage.tsx

```diff
@@ -124,6 +126,12 @@ export function InstancesPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Instances24Icon />}>Instances</PageTitle>
+        <DocsPopover
+          heading="Instances"
+          icon={<Instances16Icon />}
+          summary="Instances are virtual machines that run on Oxide."
```

Comment:
We should ask Kevin and others if there's some canonical phrasing for "on Oxide", maybe "on the Oxide platform" or "on the Oxide rack"

Reply:
“on the Oxide platform” was the consensus in the marketing channel.